src/Integration/LocalServices/ProjectSystemHelper.cs

```diff
@@ -237,7 +244,7 @@ private static IEnumerable<IVsHierarchy> EnumerateProjects(IVsSolution solution)
             }
         }
 
-        public static IEnumerable<Guid> GetAggregateProjectKinds(IVsHierarchy hierarchy)
+        private static IEnumerable<Guid> GetAggregateProjectKinds(IVsHierarchy hierarchy)
```

Comment:
Product code change, also raised a bug about this method which should not be static.